scripts/kdb.fish

```diff
@@ -241,8 +241,8 @@ function __fish_kdb_print_resolver_plugins -d 'Print a list of available resolve
 end
 
 function __fish_kdb_print_storage_plugins -d 'Print a list of available storage plugins'
-    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json ni passwd regexstore
-    set -l formats $formats simpleini simplespeclang tcl xmltool uname
+    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json mini ni passwd
+    set -l formats $formats regexstore simpleini simplespeclang tcl xmltool uname
```

Comment:
We could improve it by caching the contracts of the plugins (or even the results of `kdb list`). But it is quite an effort for tab completion of plugin names? The other parts of tab completion shouldn't be effected, or are they?

For slower computers the workaround is to install fewer plugins.